hurumap_tz/settings.py

```diff
@@ -149,5 +148,7 @@
 
 LOGGING['loggers']['hurumap_tz'] = {'level': 'DEBUG' if DEBUG else 'INFO'}
 
+HURUMAP['na_label'] = ''
```

Comment:
What's the design decision here? Why have it blank instead of the default `na_label`?

Reply:
There are datasets missing data points in districts and wards levels using N/A's make long pages on hurumap